Celeste.Mod.mm/Mod/Entities/EntityCollider.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Monocle;
+using System;
+using Microsoft.Xna.Framework;
+
+namespace Celeste.Mod.Entities {
+    [Tracked(false)]
+    public class EntityCollider<T> : Component where T : Entity {
+        public readonly string entityType = typeof(T).Name;
```

Comment:
Whats the purpose of this `entityType`/`componentType` string?

Reply:
Since all Entity Colliders are Tracked together, it makes it easier to know if a given Collider is of a given type, although I don't know if there's a more elegant solution to it, I'm willing to make it that if I get ideas.

Basically, since it's generic, it's just there for knowledge purposes

Reply:
So just for mods which type is actually tracked? In that case just provide the `Type` directly, since `type.Name` is not unique

Reply:
Ok I can do that